airflow/www/extensions/init_jinja_globals.py

```diff
@@ -42,6 +45,14 @@ def init_jinja_globals(app):
     expose_hostname = conf.getboolean('webserver', 'EXPOSE_HOSTNAME', fallback=True)
     hosstname = socket.getfqdn() if expose_hostname else 'redact'
 
+    try:
+        airflow_version = airflow.__version__
+    except Exception as e:  # pylint: disable=broad-except
+        airflow_version = None
+        logging.error(e)
+
+    git_version = get_airflow_git_version()
```

Comment:
@ashb Now that this is called for every page, [its internal logging](https://github.com/astronomer/airflow/blob/master/airflow/utils/platform.py#L60) when there is no git version has added quite a bit of noise when running the webserver—something we should consider removing?

Reply:
Yeah or just change it to log.debug to be honest.

Reply:
Updated to `log.debug`